### ISSUE TITLE
[BUG  `flutter` on your path resolves to E:\flutter\versions\stable\bin\flutter

### ISSUE THREAD

**Issue by laterdayi** [leoafarias/fvm#698]
Warning: `flutter` on your path resolves to E:\flutter\versions\stable\bin\flutter, which is not inside your
      current Flutter SDK checkout at E:\flutter\default. Consider adding E:\flutter\default\bin to the front of your
      path.

**Comment by leoafarias** [comment]
@laterdayi I will need more info on this, but it seems that this is a correct warning to the global version command.

**Comment by laterdayi** [comment]
Yes, it was a warning, and I should have dealt with that warning

**Comment by laterdayi** [comment]
I know this is a warning, but how should I address it?
 @leoafarias

**Comment by leoafarias** [comment]
You need to configure the PATH, as described in this link: https://docs.flutter.dev/get-started/install/windows/desktop?tab=download, under `Update your Windows PATH variable`, the warning provides the path you need to configure to use the global version.

**Comment by laterdayi** [comment]
`F:\flutter\default\bin`     I have configured the fvm global flutter stable and set the path, but there are still warnings

**Comment by leoafarias** [comment]
@laterdayi does the global version resolve correctly?

**Comment by Dboy233** [comment]
The issue occurs because the `fvm global` command creates a `folder shortcut`. If you add the `default` folder, which contains the global Flutter SDK directory created by `fvm`, to the `PATH` environment variable and then run the `flutter doctor` command, you will encounter the same warning message. Similarly, if you specify the SDK directory as `default` in Android Studio, there is a chance that Android Studio will crash when creating a new project. I remember that originally `Fvm` would copy the version specified by the `global` command into the `default` directory, but now it has become a `folder shortcut`.

**Comment by husainazkas** [comment]
this issue was closed but I don't see it fixed yet. I have [set global version at stable](https://fvm.app/documentation/guides/basic-commands#global) and [reroute `flutter` and `dart` command calls to FVM](https://fvm.app/documentation/guides/running-flutter#call-sdk-directly), then [set specific version in the workspace](https://fvm.app/documentation/guides/basic-commands#use) and [make sure for vscode properly configured](https://fvm.app/documentation/guides/vscode). Running `flutter doctor -v` in every terminal and everywhere shows no warning, but no for vscode flutter doctor by command palette. It shows warning like this:

```
!] Flutter (Channel stable, 3.29.0, on macOS 15.3.1 24D70 darwin-x64, locale en-ID) [1,163ms]
    • Flutter version 3.29.0 on channel stable at /Users/<username>/fvm/versions/3.29.0
    ! Warning: `flutter` on your path resolves to /usr/local/bin/flutter, which is not inside your current Flutter SDK checkout at /Users/<username>/fvm/versions/3.29.0. Consider adding /Users/<username>/fvm/versions/3.29.0/bin to the front of your path.
    ! Warning: `dart` on your path resolves to /usr/local/bin/dart, which is not inside your current Flutter SDK checkout at /Users/<username>/fvm/versions/3.29.0. Consider adding /Users/<username>/fvm/versions/3.29.0/bin to the front of your path.
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 35c388afb5 (4 weeks ago), 2025-02-10 12:48:41 -0800
    • Engine revision f73bfc4522
    • Dart version 3.7.0
    • DevTools version 2.42.2
    • If those were intentional, you can disregard the above warnings; however it is recommended to use "git" directly to perform update checks and upgrades.
```

Even I run this command palette in flutter project with no fvm configuration (which will fallback to fvm global), this error is still showing.

I've tried to add the `default` folder of fvm global into PATH, but it has no effect
